Automattic/vip-cli
# persist database data in between container restarts

## Description

We added a volume to DB so that the changes in DB get persisted between stop and start as the volume stays around.

## Steps to Test

1) Create new environment- `vip dev-env create --slug test`
2) Start it: `vip dev-env start --slug test`
3) Create a new post
4) Stop it: `vip dev-env stop --slug test`
5) Start it again: `vip dev-env start --slug test`
6) See your new post
